Resources/Prototypes/_SpaceCats/Catalog/uplink_catalog.yml

```
@@ -16,4 +16,14 @@
\ No newline at end of file
+          - SurplusBundle
+
+- type: listing
+  id: ADTUplinkInvisibleCloak
+  name: uplink-invisible-cloak-name
+  description: uplink-invisible-cloak-desc
+  productEntity: ClothingNeckInvisibleCloak
+  cost:
+    Telecrystal: 5
+  categories:
+    - UplinkWearables
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)